Sources/epoch_server/compile/epoch_player/EPOCH_server_loadPlayer.sqf

```diff
@@ -117,7 +117,11 @@ if (!isNull _player) then {
             _location = getMarkerPos "respawn_west";
             _location set[2, 0];
             if (_newLocation isEqualType [] && {(count _newLocation) == 3}) then {
-                _jammers = nearestObjects[_newLocation, ["PlotPole_EPOCH"], 6];
+				_CheckLocation = _newLocation;
```

Comment:
one thing that may need to be changed is to either add this variable to the private array at the top of the script or add private before the new variable _CheckLocation.